lmnet/configs/core/classification/lm_resnet_quantize_cifar10.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
```

Comment:
Plz write License

Reply:
fixed the license with https://github.com/blue-oil/blueoil/pull/65/commits/a7d5a5b5c0b99405dda466ecbfd0ca2d5dba56ee